net-mgmt/ipcheck-community/+POST_INSTALL

```diff
@@ -0,0 +1,4 @@
+if /usr/local/etc/rc.d/configd status > /dev/null; then
```

Comment:
Why do we need this file?

Reply:
Adding refresh of configd to +POST_INSTALL is required to register all new API endpoints that are registered by ipcheck.

I just followed the best practice at the end of a Helloworld plugin documentation: https://docs.opnsense.org/development/examples/helloworld.html

Reply:
Ok, I'll pull it, build a pkg and test. Thx :)